persistence/remote/src/main/java/org/infinispan/persistence/remote/configuration/RemoteStoreConfiguration.java

```diff
@@ -52,13 +52,15 @@ public Object readAttributeValue(String enclosingElement, AttributeDefinition at
          .immutable().build();
    static final AttributeDefinition<String> REMOTE_CACHE_NAME = AttributeDefinition.builder("remoteCacheName", "").immutable().xmlName("cache").build();
 
+   static final AttributeDefinition<String> REMOTE_SERVER_URI = AttributeDefinition.builder("remoteServerUri", "").immutable().xmlName("url").build();
```

Comment:
I would just call this `URI` or `REMOTE_URI` as it can contain multiple servers.